pennylane/tape/qnode.py

```diff
@@ -159,7 +159,7 @@ def __init__(self, func, device, interface="autograd", diff_method="best", **dif
         self.diff_options["method"] = diff_method
 
         self.dtype = np.float64
-        self.max_expansion = 2
+        self.max_expansion = 5

```

Comment:
Is that because templates are now one nesting level?

Reply:
It's because one of the `qml.inv()` tests attempts to do a multi-nested inversion 😬

Reply:
I don't want to be a pain, but if that test is very artificial would it not be better to remove it than to set a "global" variable?